rails/rubocop.yml

```diff
@@ -1,16 +1,10 @@
 require: rubocop-rails
 
-Rails/AddColumnIndex:
-  Enabled: true
-Rails/DuplicateAssociation:
-  Enabled: true
-Rails/Presence:
-  Enabled: true
-Rails/Present:
-  Enabled: true
-Rails/ReversibleMigration:
-  Enabled: true
-Rails/ReversibleMigrationMethodDefinition:
-  Enabled: true
-Rails/TransactionExitStatement:
-  Enabled: true
+Rails/I18nLocaleTexts:
```

Comment:
Initial set of rules opted out after playing a bit with `Rainforest` repo with the new configuration. 

The idea is to keep adding more rules to opt-out as we discover them to be annoying or not valuable.